packages/cli-repl/test/test-shell.ts

```diff
@@ -8,22 +8,31 @@ import stripAnsi from 'strip-ansi';
 import assert from 'assert';
 
 const PROMPT_PATTERN = /^> /m;
-const ERROR_PATTERN = /Thrown:\n([^>]*)/m;
```

Comment:
Should this go in a separate PR since it's a separate issue?

Reply:
Without that tests break when running on the compiled binary since the node version is  slightly different from the one downloaded in the CI vm